### PR TITLE
Updated commands as per go-cli and added info about new contracts

### PR DESCRIPTION
**Pull request by adi44**

### CODE REVIEW

File: docs/client.md [hunk 1]
```diff
@@ -23,37 +23,39 @@ You can use the full commands (stake) or the short form (s) as shown below.
```
Comment (ashish10677) [inline docs/client.md:23]: `create <password>` is a wrong command.
The command is `./razor create` that's all. The password is asked in the next step

Comment (ashish10677) [reply docs/client.md:23]: Also remove the warning after the create command.

File: docs/client.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Please check [Quick Start](Quick Start/Stake.md) for installation instructions
 
 ## Commands
 Run the commands in following way:
-    `node index.js <command>`
+    `./razor <command>`
 You can run following commands in CLI:
 
 `-help` See a list of available commands
```
Comment (ashish10677) [inline docs/client.md:13]: Can you separate all the commands with a heading, possibly h3, like:
### Help
`./razor --help` 
something like this

File: docs/client.md [hunk 3]
```diff
@@ -23,37 +23,39 @@ You can use the full commands (stake) or the short form (s) as shown below.
...
-Withdraw all RAZORs. Make sure RAZORs are unstaked and unlocked
+    unstake --address <address> --stakerId <staker_id> --amount <amount_of_sRZRs/RZRs> --autoWithdraw
 
-    vote|v <account> <password>
+withdraw RAZORs
 
-Start monitoring contract, commit, vote, propose and dispute automatically
+    withdraw --address <address> --stakerId <staker_id>
+    
+Delegate some RAZORs
```
Comment (ashish10677) [inline docs/client.md:42]: This is wrong. It should be activate delegation

File: docs/client.md [hunk 4]
```diff
@@ -23,37 +23,39 @@ You can use the full commands (stake) or the short form (s) as shown below.
...
 
-    transfer|t <to> <amount> <from> <password>
+    setDelegation --address <address> --status <true_or_false> --commission <commission>
 
-transfer RAZORs
+Activate Delegation
```
Comment (ashish10677) [inline docs/client.md:46]: Headings are still not correct. The heading of command should come first, then the explanation and then the command.
### Activate Delegation
The staker can choose to activate delegation if he/she likes. 
```
./razor setDelegation  --address <address> --status <true_or_false> --commission <commission_percentage>
```

File: docs/index.md [hunk 5]
```diff
@@ -20,6 +20,6 @@ Our smart contracts are hosted at [Contracts](https://github.com/razor-network/c
 Please note that the project is in active development and breaking changes are introduced frequently.
 
 ##Client
-A javascript reference client is being developed. It can be found here: [CLI](https://github.com/razor-network/cli)
+A javascript reference client is being developed. It can be found here: [CLI](https://github.com/razor-network/razor-go)
```
Comment (ashish10677) [inline docs/index.md:23]: It is a GoLang based CLI now. Not javascript.
Also, note that `##` is used carelessly in this file. Please fix those as well.

**Review by ashish10677 — CHANGES_REQUESTED** [review]

**Review by ashish10677 — APPROVED** [review]

**Review by ashish10677 — APPROVED** [review]

File: docs/Quick Start/Delegate.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 #Quick start - Delegate
 
-Razor Network in beta state allows community members to delegate their Razor tokens. You can delegate your Razor tokens to any of the validators who have enabled delegation.Razor tokens are the native tokens of Razor Network, compatible with all the ERC20 tokens.
+Razor network is a proof of stake network. In order to participate in the network as a delegator, you will need to "Delegate" your RAZORs. RAZORs are the native tokens in the network and they are compatible with the ERC20 tokens standard.
```
Comment (SkandaBhat) [inline docs/Quick Start/Delegate.md:3]: Explicity mention testnet

File: docs/index.md [hunk 7]
```diff
@@ -11,15 +11,15 @@ The core of Razor Network is a set of smart contracts, that can run on any Ether
...
-##Smart contracts
+## Smart contracts
 Our smart contracts are hosted at [Contracts](https://github.com/razor-network/contracts)
 
 Please note that the project is in active development and breaking changes are introduced frequently.
 
-##Client
-A javascript reference client is being developed. It can be found here: [CLI](https://github.com/razor-network/cli)
+## Client
+A Golang reference client is being developed. It can be found here: [CLI](https://github.com/razor-network/razor-go)
```
Comment (SkandaBhat) [inline docs/index.md:24]: Please add deployed addresses here as well

**Review by SkandaBhat — CHANGES_REQUESTED** [review]

**Pull request merged**